XamarinCommunityToolkitSample.sln

```diff
@@ -9,11 +9,11 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Xamarin.CommunityToolkit.Sa
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Xamarin.CommunityToolkit.Sample", "XamarinCommunityToolkitSample\Xamarin.CommunityToolkit.Sample.csproj", "{81AADE72-D666-4AB0-83D9-8FE366E0755E}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Xamarin.CommunityToolkit", "XamarinCommunityToolkit\Xamarin.CommunityToolkit.csproj", "{B0DCDF81-953D-47DA-A7D4-0565339BF07C}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Xamarin.CommunityToolkit", "XamarinCommunityToolkit\Xamarin.CommunityToolkit.csproj", "{B0DCDF81-953D-47DA-A7D4-0565339BF07C}"
```

Comment:
A Question here, I noticed these changes being made when I was working on UWP.  Why are these changing, and should we not be checking them in?

Reply:
I think this is a VS style thing, normally I ignore these changes but since I did modifications in `csproj` and added resources I commit these changes, for safety.